### PR TITLE
update example code of async

### PR DESCRIPTION
**Pull request by Harry-Jing**

### CODE REVIEW

File: wiki/Developer-Docs.md [hunk 1]
```diff
@@ -30,7 +30,43 @@ chatbot = Chatbot(config, conversation_id=None)
...
-```py
+```python
 ... # After the initial setup
 message = chatbot.get_chat_response("Hello world")['message']
-print(message)
\ No newline at end of file
+print(message)
+```
+
+# AsyncrevChatGPT.AsyncrevChatGPT.py
```
Comment (acheong08) [inline wiki/Developer-Docs.md:39]: They import async via `from revChatGPT.asyncChatGPT import Chatbot`

File: wiki/Developer-Docs.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ config = {
 
 chatbot = Chatbot(config, conversation_id=None)
 ```
-
+# revChatGPT.revChatGPT.py
```
Comment (acheong08) [inline wiki/Developer-Docs.md:16]: The import has to be changed. This should just be called standard functions

**Review by acheong08 — CHANGES_REQUESTED** [review]
Changes needed. Inaccurate labeling

**Review by acheong08 — APPROVED** [review]
This is better. Will publish to PyPi after some testing

**Comment by Harry-Jing** [comment]
How about writing like this
image

**Comment by acheong08** [comment]
I already changed it. https://github.com/acheong08/ChatGPT/pull/153/commits/cfe0f0a12d5d4d5bc83cd431e2782460561b3a4f

**Comment by Harry-Jing** [comment]
thanks

**Pull request merged**